examples/project-foundry/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "ponder-examples-project-foundry",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "codegen": "ponder codegen",
+    "deploy": "cd foundry && forge script script/Deploy.s.sol --broadcast --fork-url http://localhost:8545 --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80",
+    "dev": "ponder dev",
+    "generate:abi": "wagmi generate",
+    "increment": "cd foundry &&  cast send 0x5FbDB2315678afecb367f032d93F642f64180aa3 \"increment()\" --rpc-url http://localhost:8545 --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80",
+    "lint": "eslint .",
+    "start": "ponder start",
+    "start:anvil": "anvil --block-time 10",
+    "typecheck": "tsc"
+  },
```

Comment:
See monorepo comment first - after moving the ponder and foundry-specific scripts to their own directories, it might still make sense to have a top-level package.json here with certain scripts. Lmk what you think.